frontend/src/components/Map.jsx

```diff
@@ -90,8 +90,8 @@ function Map({ searchValue, results, userGeolocation }) {
 
 Map.propTypes = {
   searchValue: propTypes.string.isRequired,
-  results: propTypes.shape.isRequired,
-  userGeolocation: propTypes.shape.isRequired,
+  results: propTypes.arrayOf(propTypes.shape).isRequired,
+  userGeolocation: propTypes.objectOf(propTypes.shape).isRequired,
```

Comment:
which shape ? what are the properties inside your objects ?

see https://dev.to/cesareferrari/how-to-specify-the-shape-of-an-object-with-proptypes-3c56